package.json

```diff
@@ -37,6 +37,7 @@
     "marked": "github:mattermost/marked#5a06d1af25ca48927b9efe0d9c42c01c84e7839b",
     "mattermost-redux": "github:mattermost/mattermost-redux#9a915443ab1d695540493406926c3dd269d380fc",
     "moment-timezone": "0.5.27",
+    "p-queue": "^6.4.0",
```

Comment:
The footprint of the library small and it has good API which I intend to use in upcoming features like prefetching posts. It has API's for clearing, pausing and adding tasks dynamically too which fits perfectly for some of the upcoming features.